spec/cli_spec.rb

```diff
@@ -15,6 +15,11 @@
           -b, --disable-binding            Disable binding configuration
           -s, --specification-file FILE    Location of Cloudformation Resource Specification file
           -u, --update-specification       Update the Cloudformation Resource Specification file
+          -g RESOURCE_TYPE,RESOURCE_LOGICAL_NAME,
```

Comment:
Can you add more tests that cover the basic functionality and output of cfnlego commands?